packages/plugins/src/plugins/morpho/vaultsJob.ts

```diff
@@ -0,0 +1,48 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { getVaults } from './utils/api';
+import { Job, JobExecutor } from '../../Job';
+import { morphoVaultsCachePrefix, platformId } from './constants';
+import { Cache } from '../../Cache';
+import { tokenListInfoPrefix } from '../tokens/constants';
+import { buildTokenMetaDataItems, buildTokenPriceSources } from './helpers';
+
+const metaDataTTL = 1000 * 60 * 60 * 24 * 7; // 7 days
```

Comment:
```suggestion
const metaDataTTL = ONE_WEEK;
```
or
```suggestion
const metaDataTTL = SEVEN_DAYS;
```

We'll constantly need this kind of constants; we should define them once in a centralized location and re-use them.